Podfile

```diff
@@ -50,7 +50,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 8.4-beta'
+  pod 'WordPressKit', '~> 8.5-beta'
```

Comment:
Looks like this update caused the tests to fail due to the changes in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/611.

Adding `"ui_status": "finished",` to the JSON in `BlazeCampaignViewModelTests.swift` fixes the test.

Reply:
Thanks! I've fixed it in https://github.com/wordpress-mobile/WordPress-iOS/pull/20969/commits/21c1091325532c57b54a47df118361b290a5e636.